### PR TITLE
PAB-4449: Change info on version numbers in right drawer

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

**Comment by skom-sag** [comment]
would be part of build 25.172.0

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@dianaolives Regarding BH ticket #20, you can copy the phrasing from here :-).

File: content/streaming-analytics/troubleshooting-bundle/diagnostics-download.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ This opens the right drawer which contains a **Diagnostics** section with the fo
 - An **Enhanced diagnostics (ZIP)** link for downloading enhanced, more resource-intensive diagnostics information.
 
 It may be useful to capture this diagnostics information when experiencing problems, or for debugging EPL apps. It is also useful to provide to [product support](/additional-resources/contacting-support/) if you are filing a support ticket.
-You can find the tenant ID and version information in the **Platform info** section of the right drawer.
+You can find the tenant ID in the **Platform info** section of the right drawer.
+If you want to find out the version numbers of the different components on your tenant, click the **Download platform details** button in the **Platform info** section and then open the downloaded JSON file.
```
Comment (skom-sag) [inline content/streaming-analytics/troubleshooting-bundle/diagnostics-download.md:25]: I trust this is the same information that's also added fro cockpit/devicemanagement/administration apps ?

Comment (kailol-sag) [reply content/streaming-analytics/troubleshooting-bundle/diagnostics-download.md:25]: @skom-sag , there's general info in the topic that I linked in the next line, that is, in https://cumulocity.com/docs/get-familiar-with-the-ui/user-settings/ . But that topic still needs to be updated - I added Bugherd task 20 for this.

**Comment by kailol-sag** [comment]
Subir told me on the Teams chat that the change looks good to him.

**Review by skom-sag — APPROVED** [review]

**Pull request merged**